flutter/lib/src/integrations/native_app_start_integration.dart

```diff
@@ -10,13 +12,22 @@ class NativeAppStartIntegration extends Integration<SentryFlutterOptions> {
   NativeAppStartIntegration(
       this._frameCallbackHandler, this._nativeAppStartHandler);
 
+  @internal
+  static const integrationName = 'nativeAppStartIntegration';
```

Comment:
Should we rename this to `NativeAppStart` to match the new format, or do we need this to be like before?

Reply:
I wanted to do it all in one shot in another PR but I guess since I'm on this one it makes sense to update it here